### DISCUSSION TITLE
OPEN PEPPOL Support for ASP.NET Core and C#

### DISCUSSION THREAD

**Discussion by Sayan-Deloitte** [OxalisCommunity/Oxalis-AS4#254] [Q&A]
Hello Everyone,

I'm new to this PEPPOL Access point development and therefore seek some support in understanding whether there's any implementation/package/library available for ASP.NET platform and C#.

**Comment by Robcio35** [comment]
There was the implementation AS4.NET but the development was discontinued.

**Reply by Sayan-Deloitte** [reply]
Thanks Robert. Also, is there any document which can help us understand the interplay between Access point, SML & SMP? Specifically, how they interact, format of participant ID etc?
